lib/pages/chat/events/message_content.dart

```diff
@@ -184,6 +185,22 @@ class MessageContent extends StatelessWidget {
               final bigEmotes = event.onlyEmotes &&
                   event.numberEmotes > 0 &&
                   event.numberEmotes <= 10;
+
+              final RegExp regex = RegExp(r'https:\/\/[^\s]+');
```

Comment:
please const it

Reply:
seems like I can't const a regexp it's not a const constructor